apps/web/src/hooks/useAddWalletModal.tsx

```diff
@@ -59,18 +67,40 @@ export default function useAddWalletModal() {
         content: (
           <AddWalletModal queryRef={query} onConnectWalletSuccess={onConnectWalletSuccess} />
         ),
-        headerText: 'Connect your wallet',
+        headerText: 'Sign in or create an account',
       });
     },
     [query, showModal]
   );
 }
 
-const Container = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-
+const Container = styled(HStack).attrs({
+  gap: 24,
+  align: 'center',
+  justify: 'center',
+})`
   min-height: 280px;
   height: 100%;
+
+  display: block;
+
+  @media only screen and ${breakpoints.tablet} {
+    display: flex;
+    min-height: 280px;
+    height: 100%;
+  }
+`;
+
+const StyledWalletText = styled(TitleL)`
+  display: none;
+
+  @media only screen and ${breakpoints.tablet} {
+    display: block;
+    text-align: center;
+    font-weight: 300;
+
+    span {
+      font-style: italic;
+    }
+  }
```

Comment:
Hide the text on wallet modal for mobile view

Lmk if we still want to show it

<img width="434" alt="CleanShot 2023-09-19 at 18 08 44@2x" src="https://github.com/gallery-so/gallery/assets/4480258/ae7c0ab6-6344-48b8-9755-987c4122f9d6">

Reply:
Desktop view

<img width="1103" alt="CleanShot 2023-09-19 at 18 09 28@2x" src="https://github.com/gallery-so/gallery/assets/4480258/c5a8b7fc-aed4-4820-8cc0-98c299b0cf06">

Reply:
oo good call out, can we show it for mobile too if it fits?
something like this:
<img width="347" alt="CleanShot 2023-09-19 at 19 11 42@2x" src="https://github.com/gallery-so/gallery/assets/80802871/dd7181bf-f3fd-48c1-9d2b-99070e73dd4d">

Reply:
Sure! Updated mobile view

<img width="389" alt="CleanShot 2023-09-19 at 18 41 48@2x" src="https://github.com/gallery-so/gallery/assets/4480258/a59df840-e0ef-4b52-b7af-d61753e1997a">